test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -96,12 +96,15 @@ set(NNPA_TEST_LIST
     # ==ADDITIONAL_PARAGRAPH== NNPA has hardware limitations in dimension index size and tensor size, which are described in [NNPALimit.h](../src/Accelerators/NNPA/Conversion/ONNXToZHigh/NNPALimit.h). They are enough large for normal use cases, but if your model exceeds the limitations, CPU is used instead of NNPA.
 
     # ==OP== Add
+    # ==MIN== 14
     # ==LIM== - Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions.
```

Comment:
Just my two cents. Limitation should be per opset if we added supported opsets.

Reply:
@tungld Can you give an example of what you mean? Thanks!

Reply:
Taking Pad as an example, the document said:
```
| **Pad** |2 |18 |axes input not supported. | |
```

Limitation `axes input not supported.` can be applied to op version 18 but not 13, 11, and 2. It's because there is no `axis` input in versions 13, 11, 2: https://onnx.ai/onnx/operators/onnx__Pad.html#pad-13.

In general, different op versions have different syntax (e.g. the number/order of inputs/attributes) and different semantics. So limitation must be specific to an op version.